src/window_constants.h

```diff
@@ -87,6 +87,16 @@ enum WmMouseEvent {
 	WMME_MOVE_WINDOW,  ///< Initiate a window move.
 };
 
+/**
+ * Mouse event modes of the window manager.
+ * @ingroup window_group
+ */
+enum WmMouseEventMode {
+	WMEM_PRESS,    ///< Mouse button was pressed.
+	WMEM_REPEAT,   ///< Mouse button was held down.
```

Comment:
Minor suggestion:

```suggestion
	WMEM_REPEAT,   ///< Mouse button is held down.
```
?

(It's an internal event based on current state, rather than an external event received from the event queue.)